Paper.tex

```diff
@@ -1365,10 +1404,27 @@ \section{Block Finalisation} \label{ch:finalisation}
 The process of finalising a block involves two stages:
```

Comment:
3 stages now

Reply:
changes in [55a4099](https://github.com/ethereum/yellowpaper/pull/906/commits/55a4099d6366a2425e4c7c8ad1ba2687cd1abc89)